android/app/src/main/res/values-da/strings.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
-    <!-- Channel label for notifications shown in App > Notifications > Path Check BT -->
+  <!-- Shown on app info, app switcher, etc -->
```

Comment:
Should these values of "PathCheck BT" be included?

Reply:
I think if it is marked as non-translatable they are not used on the app. Maybe we should remove those keys all -together to avoid pulling them. I can take a look.

Reply:
If they are not used and if we don't need them, then we should remove them.
If I only remove them from Lokalise, they will get pulled in again on the next push to Lokalise. If you guys are planning to remove them from the repo, that'd be better so that the next pull from Lokalise will clean these keys out from all other language files.

Reply:
I can confirm we are not using these 2 strings, I've removed them from the code in the past but we should remove them from Lokalise too

Reply:
Thanks @juancruzgs for confirming this.